mods/cnc/maps/gdi05a/map.yaml

```diff
@@ -817,8 +817,7 @@ Rules:
 			GenericVisibility: Enemy
 			ShowOwnerRow: false
 	^Infantry:
-		SpawnViceroid:
-			Probability: 0
+		-SpawnActorOnDeath:
```

Comment:
Most of the TD missions disable Viceroid spawning by setting the `Creeps` map option to false. This regressed with this PR.